autotest/CMakeLists.txt

```diff
@@ -72,44 +75,43 @@ endfunction ()
   symlink_or_copy(${CMAKE_CURRENT_SOURCE_DIR}/conftest.py ${CMAKE_CURRENT_BINARY_DIR}/conftest.py)
   symlink_or_copy(${CMAKE_CURRENT_SOURCE_DIR}/run_slow_tests.sh ${CMAKE_CURRENT_BINARY_DIR}/run_slow_tests.sh)
 
-  if (NOT "${CMAKE_BINARY_DIR}" STREQUAL "${CMAKE_SOURCE_DIR}")
-      foreach (subdir IN ITEMS pymod proj_grids cpp/data)
-          if (SKIP_COPYING_AUTOTEST_SUBDIRS)
-            message(STATUS "Skipping copying ${CMAKE_CURRENT_SOURCE_DIR}/${subdir}")
-          else ()
-            symlink_or_copy(${CMAKE_CURRENT_SOURCE_DIR}/${subdir} ${CMAKE_CURRENT_BINARY_DIR}/${subdir})
-          endif ()
-      endforeach ()
+  SET(pytest_dirs alg benchmark gcore gdrivers gnm ogr osr pyscripts slow_tests utilities)
+
+  # Link test data/scripts into the build directory for convenience (non-Windows)
+  if (WIN32)
+      SET(python_test_path ${CMAKE_CURRENT_LIST_DIR})
+  else()
+      SET(python_test_path ${CMAKE_CURRENT_BINARY_DIR})
+      if (NOT "${CMAKE_BINARY_DIR}" STREQUAL "${CMAKE_SOURCE_DIR}")
+          foreach (subdir IN ITEMS
+                  pymod
+                  proj_grids
+                  cpp/data
+                  ${pytest_dirs})
+              if (SKIP_COPYING_AUTOTEST_SUBDIRS)
+                message(STATUS "Skipping copying ${CMAKE_CURRENT_SOURCE_DIR}/${subdir}")
```

Comment:
That can be removed. It only made sense on Windows to sometimes speed-up things to avoid repeated copying of the directories. But here we are in the branch that only symlinks.